optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/test/java/org/optaplanner/quarkus/testdata/gizmo/PrivateNoArgsConstructorSolution.java

```diff
@@ -15,15 +15,17 @@ public class PrivateNoArgsConstructorSolution {
     List<PrivateNoArgsConstructorEntity> planningEntityList;
 
     @PlanningScore
-    public final SimpleScore score;
+    public SimpleScore score;
```

Comment:
Score must not be `final`, ever. I am kinda surprised there is no fail-fast for this.